hosts/hetzci/pipelines/ghaf-nightly.groovy

```diff
@@ -23,9 +23,6 @@ def TARGETS = [
   [ target: "packages.x86_64-linux.lenovo-x1-carbon-gen11-release-installer",
     testset: null,
   ],
-  [ target: "packages.x86_64-linux.lenovo-x1-gen11-hardening-debug",
-    testset: null,
-  ],
```

Comment:
You meant to remove the target `packages.x86_64-linux.lenovo-x1-gen11-hardening-debug-installer`, right?
Here, you are removing the `packages.x86_64-linux.lenovo-x1-gen11-hardening-debug` (without -installer postfix).